### ISSUE TITLE
Why the yolov8n.xml(best.xml) Running is slow, the FPS only 2 units.

### ISSUE THREAD

**Issue by huangjiafengx** [ultralytics/ultralytics#2966]
### Search before asking

- [x] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Why the yolov8n.xml(best.xml) Running is slow which convert by this [README.md](https://github.com/ultralytics/ultralytics#readme), the FPS only 2 units?
the yolov5n.xml running is 30 fps at same environment. 
and another yolov8n.xml is 30 fps which get by the [document] (https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/230-yolov8-optimization/230-yolov8-optimization.ipynb).

![image](https://github.com/ultralytics/ultralytics/assets/92708152/9dc85de2-de62-49fc-a3a1-e2038ba87175)

My Steps for converting the  model.

as the Readme.md
1. pip install ultralytics
2. yolo predict model=yolov8n.pt source='https://ultralytics.com/images/bus.jpg'
3. Running the python script
" " "
from ultralytics import YOLO

model = YOLO("yolov8n.yaml")  # build a new model from scratch
model = YOLO("yolov8n.pt")  # load a pretrained model (recommended for training)

model.train(data="coco128.yaml", epochs=3)  # train the model
metrics = model.val()  # evaluate model performance on the validation set
results = model("https://ultralytics.com/images/bus.jpg")  # predict on an image
path = model.export(format="onnx")  # export the model to ONNX format
" " "
4. I can get files as the below.
best.bin  best.mapping  best.onnx  best.pt  last.pt
5. Getting the best.xml  by running "mo --input_model best.onnx"

_No response_

**Comment by glenn-jocher** [comment]
@huangjiafengx hello! Thanks for your question and interest in YOLOv8. There are several factors that can affect inference FPS, including hardware limitations and model complexity. Without additional information, it's difficult to pinpoint the exact reason for the difference in FPS between different YOLO models. 

However, a possible reason could be differences in model architecture and complexity between YOLOv5 and YOLOv8. YOLOv8 is designed to improve on YOLOv5 in terms of speed and accuracy, but it may require more computational power to achieve the same FPS as YOLOv5. Additionally, different conversion tools (such as those used to create the yolov8n.xml file) may also have an impact on performance.

If you're experiencing slow inference speeds with YOLOv8, there are several steps you can take to optimize performance, such as adjusting batch size or using hardware acceleration. I'd recommend taking a look at the YOLOv8 documentation and experimenting with different configurations to achieve the best performance on your specific hardware and use case. 

Please feel free to let us know if you have any other questions or concerns!
